lib/ruumba/analyzer.rb

```diff
@@ -32,14 +34,57 @@ def run(dir = ARGV)
     # @param [String] filename The filename.
     # @return [String] The extracted Ruby code.
     def extract(filename)
+      file_text, matches = parse_file(filename)
+
+      extracted_ruby = +''
```

Comment:
Why `+''` here rather than just `''`?

Reply:
Since I had added the frozen string literal pragma, strings were frozen by default. [+ unfreezes](https://ruby-doc.org/core-2.4.2/String.html#method-i-2B-40) as string. I've removed this now since the pragma is also gone.